src/dpp/slashcommand.cpp

```diff
@@ -376,9 +376,9 @@ void from_json(const nlohmann::json& j, command_data_option& cdo) {
 				break;
 			case co_attachment:
 				cdo.value = snowflake_not_null(&j, "value");
+				[[fallthrough]];
```

Comment:
this works but technically line 379 should be a `break;`, not fall-through, this is a harmless bug :)
If you can please change it to break, i'll accept it...
Thanks for the PR!